contracts/transfer/src/ops.rs

```diff
@@ -0,0 +1,13 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+// Queries
+pub const QR_BALANCE: u8 = 0x00;
+pub const QR_ROOT: u8 = 0x01;
+pub const QR_NOTES_FROM_HEIGHT: u8 = 0x02;
+
+// Transactions
+pub const TX_EXECUTE: u8 = 0x00;
```

Comment:
Since these are exported, please add some docs for the module as well as the constants or file a tech-debt issue about it.

Reply:
https://github.com/dusk-network/rusk/issues/191